pandapower/test/opf/test_pandamodels_runpm.py

```diff
@@ -729,6 +728,13 @@ def test_runpm_ploss_loading():
 
     assert net.res_trafo.pl_mw.values.sum() < net_org.res_trafo.pl_mw.values.sum()
 
+    ### test loading reduction with Q-optimierung
+    net = deepcopy(net_org)
+    pp.runpm_loading(net)
+    assert np.all(net.res_line.loading_percent.values <= net_org.res_line.loading_percent.values)
```

Comment:
Is this a reasonable check? 
- this checks if the loading of every line is lower or at least the same as before